packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -44,6 +44,14 @@ import { IInterval, IntervalConflictResolver, IntervalTree, IntervalNode } from
 
 const reservedIntervalIdKey = "intervalId";
 
+export const IntervalOpType = {
+	ADD: "add",
+	DELETE: "delete",
+	CHANGE: "change",
+	PROPERTYCHANGE: "propertyChange",
```

Comment:
This is pre-existing, but I do think it would be more readable and match the camel case better to use SCREAMING_SNAKE case rather than just uppercase.
```suggestion
	PROPERTY_CHANGE: "propertyChange",
```